### PR TITLE
Triage meetings have moved to Zoom

### PR DESCRIPTION
**Pull request by peternied**
### Description
Triage meetings have moved to Zoom

### Related Issues
- Related #13057 

### Check List
- [ ] ~New functionality includes testing.~
  - [ ] ~All tests pass~
- [X] New functionality has been documented.
  - [ ] ~New functionality has javadoc added~
- [X] Failing checks are inspected and point to the corresponding known issue(s) (See: [Troubleshooting Failing Builds](../blob/main/CONTRIBUTING.md#troubleshooting-failing-builds))
- [X] Commits are signed per the DCO using --signoff
- [ ] ~Commit changes are listed out in CHANGELOG.md file (See: [Changelog](../blob/main/CONTRIBUTING.md#changelog))~
- [X] Public documentation issue/PR [created](https://github.com/opensearch-project/documentation-website/issues/new/choose)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Improve built-in secure transports support ([#12907](https://github.com/opensearch-project/OpenSearch/pull/12907))
 - Update links to documentation in rest-api-spec ([#13043](https://github.com/opensearch-project/OpenSearch/pull/13043))
 - Refactoring globMatch using simpleMatchWithNormalizedStrings from Regex ([#13104](https://github.com/opensearch-project/OpenSearch/pull/13104))
+- Triage meetings have moved to Zoom ([#13208](https://github.com/opensearch-project/OpenSearch/pull/13208))
```
Comment (kotwanikunal) [inline CHANGELOG.md:44]: I don't think we need this update in the product release notes :)

File: TRIAGING.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Each meeting we seek to address all new issues. However, should we run out of ti
 
 ### How do I join the Triage meeting?
 
-Meetings are hosted regularly at 10:00a - 10:55a Central Time every Wednesday and can be joined via [Chime](https://aws.amazon.com/chime/), with this [meeting link](https://chime.aws/1988437365).
+Meetings are hosted regularly at 10:00a - 10:55a Central Time every Wednesday and can be joined via [Zoom](https://zoom.us/download), with this [meeting link](https://us02web.zoom.us/j/86287450465) and passcode `805212`.
 
 After joining the Chime meeting, you can enable your video / voice to join the discussion.  If you do not have a webcam or microphone available, you can still join in via the text chat.
```
Comment (owaiskazi19) [inline TRIAGING.md:19]:
```suggestion
After joining the Zoom meeting, you can enable your video / voice to join the discussion.  If you do not have a webcam or microphone available, you can still join in via the text chat.
```

**Review by kotwanikunal — APPROVED** [review]

**Review by VachaShah — APPROVED** [review]

**Comment by owaiskazi19** [comment]
@peternied do we need to backport this to 2.x?

**Comment by peternied** [comment]
Thanks for checking, doesn't seem relevant since its project process that isn't associated with any code / assets directly.  Furthermore the file was never backported in the first place https://github.com/opensearch-project/OpenSearch-1/blob/2.x/TRIAGING.md

**Pull request merged**